# Now deactivating MTD support if a non MCD capable CAM is inserted after removing
# a previously used CAM that is MCD capable.
#
--- a/ci.c	2018/03/19 16:37:03	5.0
+++ b/ci.c	2018/09/23 10:17:20
@@ -1213,6 +1213,7 @@
         }
      else {
         dsyslog("CAM %d: doesn't reply to QUERY - only a single channel can be decrypted", CamSlot()->SlotNumber());
+        CamSlot()->MtdActivate(false);
         state = 4; // normal operation
         }
      }
